csharp/sdk/4.0/methods.cs

```diff
@@ -3746,7 +3753,7 @@ namespace = SdkUtils.EncodeParam(namespace);
   /// You can use this function to change the string and integer properties of
   /// a dashboard. Nested objects such as filters, dashboard elements, or dashboard layout components
   /// cannot be modified by this function - use the update functions for the respective
-  /// nested object types (like [update_dashboard_filter()](#!/4.0/Dashboard/update_dashboard_filter) to change a filter)
+  /// nested object types (like [update_dashboard_filter()](#!/3.1/Dashboard/update_dashboard_filter) to change a filter)
```

Comment:
need a ticket to fix this internally but it can wait until 24.0

Reply:
Should I bother manually changing these for 23.20 or just let it sit since it's just comments?

Reply:
no, we should change the source to correct this for 24.0. We just need to make sure we have a ticket to keep track.